testbook/client.py

```diff
@@ -105,8 +105,13 @@ def execute_cell(self, cell, **kwargs) -> Union[Dict, List[Dict]]:
             try:
                 cell = super().execute_cell(self.nb['cells'][idx], idx, **kwargs)
             except CellExecutionError as e:
-                # TODO: drop usage of eval
-                raise eval(e.ename)(e) from None
+                # Look for in-built Python exception
+                for klass in all_subclasses(Exception):
+                    if klass.__name__ == e.ename:
+                        raise klass(e)
```

Comment:
You likely also want to try to preserve the stack trace ... unfortunately this gets pretty complicated: https://unterwaditzer.net/2018/python-custom-tracebacks.html as the primitives you need can't be constructed explicitly so you have to fake reconstruction of the objects :/ Maybe another TODO for that in the future

Reply:
This makes me inclined to instead always raise a `TestbookRuntimeException` class extending the `CellExecutionError` but with an associated `.nbexception` attribute that has the translated exception for users to reuse if we can't consistently reraise with a real stacktrace.

Thoughts?

Reply:
Sure, that makes sense.. only thing is that it would make the error assertions non-intuitive. 

Instead of `raise klass(e.evalue)`, we could do `raise klass from e` - that way it preserves the stack trace?

Reply:
`raise klass(e.evalue) from e` is what you want